zstackctl/zstackctl/ctl.py

```
@@ -1647,6 +1647,12 @@ def find_process_by_cmdline(keyword):
 
     return None
 
+
+def _bracket_ipv6(ip):
+    """IPv6 地址在 URL / SSH 命令中需要用方括号包裹，IPv4 原样返回。"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)